schemas/nl.basjes/yauaa_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a context generated by the YAUAA enrichment after parsing the user agent",
+    "self": {
+        "vendor": "nl.basjes",
+        "name": "yauaa_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "DeviceClass": {
+            "description": "Examples: Desktop, Mobile, Tablet, Phone, TV, Unknown. See https://yauaa.basjes.nl/README-Output.html",
+            "type": "string"
+        },
+        "DeviceName": {
+            "description": "Example: Google Nexus 6",
+            "type": "string"
+        },
+        "DeviceBrand": {
+            "description": "Example: Google",
+            "type": "string"
```

Comment:
are there properties which are nullable?

Reply:
All the properties that are not in `required` (i.e. all of them except `DeviceClass`) will not be in the context if they don't have a value. Is it what nullable means?

Reply:
no, it means the sdk can send back `null` for one of those field

Reply:
Ok then no it should never happen.